proto/v1beta1/vulnerability_go_proto/vulnerability.pb.go

```diff
@@ -379,8 +379,10 @@ type Vulnerability_WindowsDetail_KnowledgeBase struct {
 func (m *Vulnerability_WindowsDetail_KnowledgeBase) Reset() {
 	*m = Vulnerability_WindowsDetail_KnowledgeBase{}
 }
-func (m *Vulnerability_WindowsDetail_KnowledgeBase) String() string { return proto.CompactTextString(m) }
```

Comment:
I'm not sure why this is formatting slightly differently for me. Should I manually revert the changes to this file?

Reply:
This is strange, yes. Go formatting doesn't really restrict the number of chars per line. Let's leave this as is and come back if it starts causing issues, which I don't think it will. Thanks for flagging this!

Reply:
Also, swagger sometimes just changes the code it generates or the formatting, so you may see a similar thing in the future. It's usually fine to let it do its thing.:)